topcoffea/modules/event_selection.py

```diff
@@ -79,3 +79,26 @@ def get_Z_peak_mask(lep_collection,pt_window,flavor="os",zmass=91.2):
         raise Exception(f"Error: flavor requirement \"{flavor}\" is unknown.")
     sfosz_mask = ak.flatten(ak.any((zpeak_mask & sf_mask),axis=1,keepdims=True)) # Use flatten here because it is too nested (i.e. it looks like this [[T],[F],[T],...], and want this [T,F,T,...]))
     return sfosz_mask
+
+# Returns a mask for all events with any os lepton pair within low region of off-Z
+# Mask will be True if any combination of 2 leptons from within the given collection satisfies the requirement
+def get_off_Z_mask_low(lep_collection,pt_window,flavor="os"):
+    ll_pairs = ak.combinations(lep_collection, 2, fields=["l0","l1"])
+    zpeak_mask = ((91.2-(ll_pairs.l0+ll_pairs.l1).mass)>pt_window)
+    if flavor == "os":
+        sf_mask = (ll_pairs.l0.pdgId == -ll_pairs.l1.pdgId)
+    elif flavor == "ss":
+        sf_mask = (ll_pairs.l0.pdgId == ll_pairs.l1.pdgId)
+    elif flavor == "as": # Same flav any sign
+        sf_mask = ((ll_pairs.l0.pdgId == ll_pairs.l1.pdgId) | (ll_pairs.l0.pdgId == -ll_pairs.l1.pdgId))
+    else:
+        raise Exception(f"Error: flavor requirement \"{flavor}\" is unknown.")
+    sfosz_mask = ak.flatten(ak.any((zpeak_mask & sf_mask),axis=1,keepdims=True)) # Use flatten here because it is too nested (i.e. it looks like this [[T],[F],[T],...], and want this [T,F,T,...]))
+    return sfosz_mask
+
+# Returns a mask for all events with any os lepton pair
```

Comment:
This looks like it is returning a mask for all events with any _same-flavor_ OS pair, right? If so, I would suggest updating the comment to avoid confusion down the road (if someone reads the description and not the whole function).

Reply:
Sure, I have updated it. Thanks for the suggestion!